learnEcl/2800-module.md

```diff
@@ -2,3 +2,121 @@
 title: Module
 slug: module
 ---
+
+# MODULE
+
+MODULE is a fundamental organizational unit that contains one or more functions, actions, or definitions. Modules are used to group together code that serves a common purpose, making it easier to manage and reuse code components. 
+
+## Notes
+* OUTPUTs can not be used within a module.
+* For modules to be called/used from outside the current module, EXPORT is required.
+* A Module's name should match the file name it's in. If not "Error: Definition must contain EXPORT or SHARED " is generated.
```

Comment:
Technically, a .ecl file (that is not a BWR) must contain exactly one EXPORTed attribute and that attribute must be named the same as the filename.  You can have non-EXPORT attributes prior to the EXPORTed attribute in the file.